build.sh

```diff
@@ -28,10 +29,11 @@ function print_help() {
   echo "-t           create a portable package (experimental, Linux only)"
   echo "-s           update submodules"
   echo "-b           custom build properties file"
+  echo "-S           skip fetching submodules"
```

Comment:
Since it skips downloading external libraries as well, I think that the message should be `skip fetching submodules and additional resources`

Reply:
I understand that `-S` is an opposite of `-s`, but perhaps `--skip-fetch` would be more readable?

Reply:
Flag changed as requested.